### ISSUE TITLE
Clarification of 'Export to HTML with Attachments'

### ISSUE THREAD

**Issue by adamphetamine** [thundernest/import-export-tools-ng#26]
I'm pretty sure I am having this issue-
https://github.com/thundernest/import-export-tools-ng/issues/22

But I'm having some trouble understanding the solution.
To explain- 
When I use the Import / Export Tools NG to export a folder 
'Export to HTML with Attachments'

If there is no emails in that folder, but there is subfolders that include emails, nothing is exported.
If there are emails in that folder, and also subfolders containing emails, it exports the emails in the first level of the folder structure, and ignores all subfolders

I'm guessing that the answer here is to make the folder structure simple and not have subfolders, but in the longer term please consider a feature request to be able to export and entire mail account including subfolders to HTML.

Setup
Mac OS v10.14.6
Thunderbird 68.2.0
Import/ Export Tools NG v4.0.4

In conclusion- thank you for your work, I'm blown away that this tool is even available.

**Comment by cleidigh** [comment]
@adamphetamine 
Your observation is correct.  As in the original Import-exportTools extension, only the export
mbox includes the sub folders structure.  You are also not the first to request this.
It absolutely makes sense to add this.  I need to update other extensions first, but I think this feature
is probably near the top of the list for enhancements.
please stand by...
Thanks

**Comment by adamphetamine** [comment]
Cheers @cleidigh 
Thanks for the extra info. I'll be happy to make a donation for this feature and to help test.
Thanks again for your swift and helpful reply.

**Comment by cleidigh** [comment]
@adamphetamine 
If I could make this happen after updating my other extensions that would be great.  Thunderbird can always use donations.  I am just doing this for the community.
FYI I am going to close this and create an enhancement issue to track.  I will flag you there so you will get updates.
cheers / thanks
